### ISSUE TITLE
ForbiddenModule doesn't detect grouped aliases

### ISSUE THREAD

**Issue by hoyon** [rrrene/credo#946]
### Environment

* Credo version (`mix credo -v`): 1.6.2
* Erlang/Elixir version (`elixir -v`): 1.13.2
* Operating system: Linux

### What were you trying to do?

Use `Credo.Check.Warning.ForbiddenModule` to prevent certain modules being used in our codebase.

For example this test case:
```elixir
  test "it should report on grouped aliases" do
    """
    defmodule CredoSampleModule do
      alias CredoSampleModule.{ForbiddenModule}
      def some_function, do: ForbiddenModule.another_function()
    end
    """
    |> to_source_file
    |> run_check(@described_check, @opts)
    |> assert_issue()
  end
```

### Expected outcome

An issue is flagged as `CredoSampleMoudle.ForbiddenModule` has been used.

### Actual outcome

No issue flagged.

---

It is quite common for the modules we want to ban to be used like this so this check is currently not particularly useful for us.

Happy to put in a PR when I have some time but I'm not very familiar with the internals of credo.
